### PR TITLE
Targets fixes

### PR DESCRIPTION
**Pull request by mwasilew**

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 3033581cfd795e5d6bbc23e29f815dbf3c2f9954 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/751/docs/artifacts/html/index.html

File: source/getting-started/install-fioctl/index.rst [hunk 1]
```diff
@@ -172,6 +172,29 @@ authentication process.
...
+
+        export FIOCTL_FACTORY=<factory>
+        fioctl targets list
+
+   *  config file's factory option
+
+     .. code-block:: shell
+
+        echo "factory: <factory>" >> $HOME/.config/fioctl.yaml
+        fioctl targets list
```
Comment (vkhoroz) [inline source/getting-started/install-fioctl/index.rst:196]: This is awesome!
@doanac is it how you'd like the user to percieve this ?

**Review by vkhoroz — COMMENTED** [review]
LGTM

File: source/getting-started/install-fioctl/index.rst (hunk 1 above)
Comment (doanac) [reply source/getting-started/install-fioctl/index.rst:196]: yes. I like this approach more.

Comment (MatthewCroughan) [reply source/getting-started/install-fioctl/index.rst:196]: Perfect.

**Review by MatthewCroughan — APPROVED** [review]
Exactly the right way to do it imo. :)

**Pull request merged**